contentTypesRouter.js

```diff
@@ -0,0 +1,190 @@
+const path = require("path");
+const fs = require("fs").promises;
+const express = require("express");
+const { orderBy, slice } = require("lodash");
+const gitUtils = require("./git-utils");
+const utils = require("./utils/common");
+const media = require("./utils/media");
+
+const { getContentType } = utils;
+
+module.exports = (root, gitRepositoryRoot, publicBaseUrl) => {
```

Comment:
isn't `root` and `gitRepositoryRoot` are the same folders?

Reply:
Root is for `content` `media` etc folders, while gitRoot is git repository root

Reply:
rename: root => dbDir, gitRepositoryRoot => repositoryRoot